Doc/c-api/gcsupport.rst

```diff
@@ -60,6 +60,26 @@ Constructors for container types must conform to two rules:
    followed by the :c:member:`~PyTypeObject.tp_traverse` handler become valid, usually near the
    end of the constructor.
 
+.. c:function:: int PyObject_GC_IsTracked(PyObject *op)
+
+   Returns 1 if the object *op* is being currently tracked by the garbage
+   collector and 0 otherwise.
+
+   This function will return 0 for objects that do not have implemented support
+   for the garbage collector.
```

Comment:
I'm confused by this sentence. Maybe try to merge it with the previous one? Like:

"Returns 1 if the object type implements the GC protocol and the object *op* is being currently tracked ..."

Same remark for the other doc function.